### PR TITLE
[UIK-2382][feature-popover] added some accessibility improvements

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
I've changed some aria-* attributes and added autoFocus by default.

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Review by ksimanov — APPROVED** [review]

File: semcore/feature-popover/src/FeaturePopover.jsx [hunk 1]
```diff
@@ -132,15 +129,15 @@ class FeaturePopoverPopper extends Component {
     } = this.asProps;
 
     return sstyled(styles)(
-      <Popper.Popper disableEnforceFocus zIndex={zIndex}>
+      <Popper.Popper disableEnforceFocus zIndex={zIndex} role={'dialog'} tabIndex={0} autoFocus>
```
Comment (sheila-semrush) [inline semcore/feature-popover/src/FeaturePopover.jsx:132]: is this the right node for role=dialog and so on? 
I should be able to define an aria-label for the popper, but when I do this:
`<FeaturePopover.Popper aria-label='New feature: Export'`
aria-label goes to another node, not the dialog
image
or should I set the aria-label to another element?

File: semcore/feature-popover/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.48.0] - 2024-10-30
+
+### Changed
+
+- Improve accessibility for component.
```
Comment (sheila-semrush) [inline semcore/feature-popover/CHANGELOG.md:9]:
```suggestion
- A11y improvements: added role="dialog" and autofocus for the popover, made an aria-label or aria-labelledby a required attribute. Disabled aria-haspopup and changed the default interaction to "none" for the trigger.
```
what do you think about a bit more detailed changelog?

File: website/docs/components/feature-popover/examples/animation_example.tsx [hunk 3]
```diff
@@ -9,12 +9,12 @@ const Demo = () => {
   const handleVisibleChange = (visible) => () => setVisible(visible);
 
   return (
-    <FeaturePopover visible={visible} onVisibleChange={setVisible} disablePortal>
```
[the remark this answers is not in the document]
Comment (ilyabrower) [reply website/docs/components/feature-popover/examples/animation_example.tsx:12]: returned back "disablePortal"

File: semcore/feature-popover/src/FeaturePopover.jsx (hunk 1 above)
Comment (ilyabrower) [reply semcore/feature-popover/src/FeaturePopover.jsx:132]: No, I did the role incorrectly, I corrected it

**Review by sheila-semrush — APPROVED** [review]

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**